src/EPR.Payment.Mopup.Common.Data/AppDbContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using EPR.Payment.Mopup.Common.Data.DataModels.Lookups;
+using EPR.Payment.Mopup.Common.Data.Interfaces;
+using Microsoft.EntityFrameworkCore;
+using System.Diagnostics.CodeAnalysis;
+
+namespace EPR.Payment.Mopup.Common.Data
+{
+    [ExcludeFromCodeCoverage]
```

Comment:
If this needs to be ignore then at least add Justification

Reply:
I think it can be excluded from code coverage as it does not add any value to the tests. The focus of the tests is business logic and this is boilerplate code. Happy to discuss if you disagree

Reply:
Not necessary to add unit test for here